src/reducers/ast.js

```diff
@@ -221,8 +221,10 @@ export function getPausePoint(
     return;
   }
 
-  const linePoints = pausePoints[line];
-  return linePoints && linePoints[column];
+  const linePoints = pausePoints[String(line)];
+  if (linePoints && column != null) {
```

Comment:
I suppose `linePoints && column` would not be clear enough?

Reply:
That'd work if you prefer (assuming `column` is always non-zero).